example/CountlyRNExample/Events.tsx

```diff
@@ -4,70 +4,30 @@ import { SafeAreaView } from "react-native-safe-area-context";
 import Countly from "countly-sdk-react-native-bridge";
 import CountlyButton from "./CountlyButton";
 
-interface Segmentation {}
-interface SegmentationCustom_1 extends Segmentation {
-    Country: string;
-    Age: string;
-}
-interface EventProps {
-    eventName: string;
-    segments?: Segmentation;
-    eventCount?: number;
-    eventSum?: string;
-}
-interface EventPropsCustom_1 extends EventProps {
-    segments?: SegmentationCustom_1;
-}
-
 const basicEvent = () => {
     // example for basic event
-    const event = { eventName: "Basic Event", eventCount: 1 };
-    Countly.sendEvent(event);
+    Countly.events.recordEvent("Basic Event", undefined, 1);
 };
```

Comment:
add old code to eventsLegacy file and add a button for it

Reply:
done